### DISCUSSION TITLE
Support for Vault banks

### DISCUSSION THREAD

**Discussion by Bloodlex** [EssentialsX/Essentials#4513] [Ideas and Feature Suggestions]
Hello 😊,

I would like to ask if it is possible to add support for Vault banks and not only player accounts. It would be a great addition to my server, because now we use for this a "player" account which is basically not a player, however it isn't designed for players to have for instance shared balance.

I would be very grateful if you could consider to add support for that, because many of my players would like to have shared balance and run a company together. Thank you very much.

**Comment by JRoy** [comment]
Could you expand on how you'd like banks to be supported, currently there is very little support for banks in most economy plugins and I don't know how you'd imagine bank support in Essentials.
